src/components/SuperDrawer/SuperDrawer.stories.tsx

```diff
@@ -368,55 +377,70 @@ function TestDrawerContent({ book, hasActions = true }: TestDrawerContentProps)
   }
 
   return (
-    <SuperDrawerContent
-      actions={
-        hasActions
-          ? [
-              { label: "Close", onClick: () => closeDrawer(), variant: "tertiary" },
-              { label: `Purchase "${book.bookTitle}"`, onClick: handlePurchase },
-            ]
-          : undefined
-      }
-    >
-      <h2 css={Css.xlSb.mb1.$}>{book.bookTitle}</h2>
-      <p css={Css.base.$}>
-        <strong>Description:</strong> {book.bookDescription}
-      </p>
-      <hr css={Css.my1.$} />
-      <div css={Css.df.gap1.aic.$}>
-        <p>
-          <strong>Author:</strong> {book.authorName}
+    <>
+      <SuperDrawerHeader hideControls={hideControls}>
+        <div css={Css.df.jcsb.aic.gap2.$}>
+          <div css={Css.fg1.df.aic.gap2.$}>
```

Comment:
:thinking: what do you think about some sort of `TableActions` type component that, even within a `SuperDrawerHeader` gives the user some higher-level left/right placeholders to fill in, so that we don't have the h1/gap2/etc. padding copy/pasted across each usage site?

Granted, having the escape hatch for "let me do whatever" is probably necessary too.

...like maybe `SuperDrawerHeader title={...} left={...} right={...}`, which if set use the default/out-of-box padding/spacing, but if title/left/right are not used, then we just do whatever children is?

Reply:
That is an interesting idea. Would help with some default styling/positioning. Maybe two potential interfaces, one with children, and then one with title/left/right, where title is required... 🤔

Reply:
Alrighty - just gave that a shot. Let me know what you think.